stix_shifter_modules/darktrace/stix_transmission/connector.py

```diff
@@ -89,6 +89,10 @@ async def create_results_connection(self, search_id, offset, length):
             return_obj['data'] = results[offset:length]
             return_obj = self.check_empty_data(return_obj)
 
+        except json.decoder.JSONDecodeError as ex:
+            response_dict['code'] = 1001
```

Comment:
1001 is authentication_fail error code, but maybe we need 'service_unavailable' here?